packages/fast-components-styles-msft/src/utilities/elevation.ts

```diff
@@ -121,20 +125,18 @@ export function elevationShadow(
             yOffsetMultiplier,
             opacity,
             blurMultiplier,
+            blurBase,
```

Comment:
is the elevationShadow exposed from this package?

Reply:
Yes, looks like all exported functions in this file are exported from the package.

Reply:
In that case if anyone is using it, `blurBase` will be `undefined` and `elevationShadow()` will not return the right thing, can you add a default 0 for `blurBase`?